src/styles/generic/_links.scss

```diff
@@ -34,12 +34,14 @@ a,
   a,
   :link,
   :visited {
-    text-decoration: underline;
-
-    &:focus,
-    &:active,
-    &:hover {
-      text-decoration: none;
+    &:not(.w-chip) {
```

Comment:
hmmm I feel weird defining this here. Is it possible for the chips.scss file to own its own overrrides?